samples/net/rpl_border_router/README.rst

```diff
@@ -0,0 +1,169 @@
+IEEE 802.15.4 network but Bluetooth IPSP network functionality is also possible.
+
+The source code for this sample application can be found at:
+:file:`samples/net/rpl_border_router`.
+
+Requirements
+************
+
+- Real device like Freedom Board (FRDM-K64F) with MCR20A IEEE 802.15.4 support.
+- Linux machine with web browser and the screen terminal emulator (optional).
+- Ethernet access for management purposes (optional).
+
+Note that there is no support for running RPL border router in QEMU, as the
+border router requires access to a real radio network technology like
+IEEE 802.15.4 which is not available in QEMU.
+
+Building and Running
+********************
+
+By default, the integrated HTTP server is configured to listen at port 80.
+If you want to modify the HTTP server config options, please check
+the configuration settings in :file:`samples/net/rpl_border_router/src/main.c`
+file and also in the :file:`samples/net/rpl_border_router/src/config.h` file.
+
+This sample code supports both static and dynamic (DHCPv4) IP addresses that
+can be defined in the project configuration file:
+
+.. code-block:: console
+
+	CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::1"
+	CONFIG_NET_APP_MY_IPV4_ADDR="192.0.2.1"
+
+Note that the IPv4 address is only used for connection to the integrated web
+server that provides admin web page for management purposes. The web server
+can also be connected using IPv6 address. If you do not have a management
+interface, then IPv4 support can be disabled in configuration file by setting
+:option:`CONFIG_NET_IPV4` to "n". The RPL router uses only IPv6 when routing
+the network traffic.
+
+Note that the default project configuration file
+:file:`samples/net/rpl_border_router/prj.conf` does not currently provide
+a working system as there is no boards that would provide suitable network
+interface support. The prj.conf file is provided only to compile test the
+border router sample application.
+
+It is possible to use the border router application with these boards and
+add-on cards:
+
+* FRDM-K64F with TI CC2520 card. See
+  https://wiki.zephyrproject.org/index.php?title=TI_CC2520
+  for instructions for wiring etc.
+
+* FRDM-K64F with Freescale CR20A card.
+  http://www.nxp.com/products/developer-resources/hardware-development-tools/freedom-development-boards/freedom-development-board-for-mcr20a-wireless-transceiver:FRDM-CR20A
+
+You can build the application like this for CC2520
+
+.. code-block:: console
+
+	$ make pristine
+	$ make CONF_FILE=prj_frdm_k64f_cc2520.conf BOARD=frdm_k64f flash
+
+and for CR20A like this
+
+.. code-block:: console
+
+	$ make pristine
+	$ make CONF_FILE=prj_frdm_k64f_mcr20a.conf BOARD=frdm_k64f flash
+
+By default, the RPL border router application enables net shell support and
+provides some useful commands for debugging and viewing the network status.
+
+The **br repair** command will cause the RPL network to re-configure itself.
```

Comment:
I wanted to keep the command short here thus "br" was used. Note that this command is only meant/used/available for this border router application so user probably understand what the command is for.

Reply:
> probably understand what the command is for.

Probably. I'd approach it from the other side though: "can it cause confusion or not?" Arguably it can. If short prefix is desired, I'd suggest "brt".